{"schemaVersion":{"major":0,"minor":3,"patch":0},"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/passwordandnfinityframework\/nfinityclouderror"]}],"hierarchy":{"paths":[["doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/documentation\/PasswordAndNfinityFramework"]]},"topicSections":[{"title":"Enumeration Cases","identifiers":["doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/documentation\/PasswordAndNfinityFramework\/NfinityCloudError\/attemptExceeded","doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/documentation\/PasswordAndNfinityFramework\/NfinityCloudError\/cloudError(errorCode:errorMessage:)","doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/documentation\/PasswordAndNfinityFramework\/NfinityCloudError\/cloudHTTPError(httpCode:response:)","doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/documentation\/PasswordAndNfinityFramework\/NfinityCloudError\/httpDeserialization(httpCode:cause:headers:)","doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/documentation\/PasswordAndNfinityFramework\/NfinityCloudError\/network(cause:)"]},{"generated":true,"title":"Default Implementations","identifiers":["doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/documentation\/PasswordAndNfinityFramework\/NfinityCloudError\/Error-Implementations"]}],"sections":[],"relationshipsSections":[{"kind":"relationships","type":"conformsTo","identifiers":["doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/s5ErrorP","doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/s8SendableP"],"title":"Conforms To"}],"abstract":[{"text":"Represents an enum hierarchy for errors related to the NfinityÂ Cloud.","type":"text"}],"metadata":{"navigatorTitle":[{"text":"NfinityCloudError","kind":"identifier"}],"modules":[{"name":"PasswordAndNfinityFramework"}],"title":"NfinityCloudError","fragments":[{"text":"enum","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"NfinityCloudError"}],"roleHeading":"Enumeration","role":"symbol","externalID":"s:27PasswordAndNfinityFramework0C10CloudErrorO","symbolKind":"enum"},"identifier":{"url":"doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/documentation\/PasswordAndNfinityFramework\/NfinityCloudError","interfaceLanguage":"swift"},"primaryContentSections":[{"declarations":[{"tokens":[{"kind":"keyword","text":"enum"},{"text":" ","kind":"text"},{"kind":"identifier","text":"NfinityCloudError"}],"languages":["swift"],"platforms":["Mac Catalyst"]}],"kind":"declarations"}],"kind":"symbol","references":{"doc://com.aheaditec.widgetizing.PasswordAndNfinityFramework/documentation/PasswordAndNfinityFramework/NfinityCloudError/network(cause:)":{"url":"\/documentation\/passwordandnfinityframework\/nfinityclouderror\/network(cause:)","kind":"symbol","type":"topic","title":"NfinityCloudError.network(cause:)","identifier":"doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/documentation\/PasswordAndNfinityFramework\/NfinityCloudError\/network(cause:)","fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"text":"network","kind":"identifier"},{"kind":"text","text":"("},{"text":"cause","kind":"externalParam"},{"kind":"text","text":": "},{"text":"CommunicatorError","kind":"typeIdentifier","preciseIdentifier":"s:21CommunicatorFramework0A5ErrorO"},{"text":")","kind":"text"}],"role":"symbol","abstract":[{"type":"text","text":"Case represents network error"}]},"doc://com.aheaditec.widgetizing.PasswordAndNfinityFramework/documentation/PasswordAndNfinityFramework/FaceDetectionSender/sendFaceDetection(imageData:)":{"title":"sendFaceDetection(imageData:)","kind":"symbol","abstract":[{"text":"Method for sending face detection data, and returns an ","type":"text"},{"type":"codeVoice","code":"Swift.Result"},{"text":" instance.","type":"text"}],"url":"\/documentation\/passwordandnfinityframework\/facedetectionsender\/sendfacedetection(imagedata:)","role":"symbol","identifier":"doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/documentation\/PasswordAndNfinityFramework\/FaceDetectionSender\/sendFaceDetection(imageData:)","type":"topic","fragments":[{"kind":"keyword","text":"func"},{"text":" ","kind":"text"},{"text":"sendFaceDetection","kind":"identifier"},{"text":"(","kind":"text"},{"kind":"externalParam","text":"imageData"},{"text":": ","kind":"text"},{"kind":"typeIdentifier","text":"Data","preciseIdentifier":"s:10Foundation4DataV"},{"text":") ","kind":"text"},{"kind":"keyword","text":"async"},{"text":" -> Swift","kind":"text"},{"text":".","kind":"text"},{"text":"Result","preciseIdentifier":"s:s6ResultO","kind":"typeIdentifier"},{"kind":"text","text":"<"},{"kind":"typeIdentifier","preciseIdentifier":"s:27PasswordAndNfinityFramework18FaceDetectionStateO","text":"FaceDetectionState"},{"kind":"text","text":", "},{"kind":"typeIdentifier","preciseIdentifier":"s:27PasswordAndNfinityFramework0C10CloudErrorO","text":"NfinityCloudError"},{"kind":"text","text":">"}]},"doc://com.aheaditec.widgetizing.PasswordAndNfinityFramework/documentation/PasswordAndNfinityFramework/NfinityCloudError/Error-Implementations":{"url":"\/documentation\/passwordandnfinityframework\/nfinityclouderror\/error-implementations","type":"topic","kind":"article","title":"Error Implementations","identifier":"doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/documentation\/PasswordAndNfinityFramework\/NfinityCloudError\/Error-Implementations","role":"collectionGroup","abstract":[]},"doc://com.aheaditec.widgetizing.PasswordAndNfinityFramework/documentation/PasswordAndNfinityFramework":{"title":"PasswordAndNfinityFramework","identifier":"doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/documentation\/PasswordAndNfinityFramework","type":"topic","role":"collection","url":"\/documentation\/passwordandnfinityframework","kind":"symbol","abstract":[]},"doc://com.aheaditec.widgetizing.PasswordAndNfinityFramework/documentation/PasswordAndNfinityFramework/NfinityCloudError/httpDeserialization(httpCode:cause:headers:)":{"url":"\/documentation\/passwordandnfinityframework\/nfinityclouderror\/httpdeserialization(httpcode:cause:headers:)","abstract":[{"type":"text","text":"Indicates a problem with deserializing the HTTP response into a model object."}],"title":"NfinityCloudError.httpDeserialization(httpCode:cause:headers:)","kind":"symbol","role":"symbol","identifier":"doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/documentation\/PasswordAndNfinityFramework\/NfinityCloudError\/httpDeserialization(httpCode:cause:headers:)","fragments":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"text":"httpDeserialization","kind":"identifier"},{"kind":"text","text":"("},{"text":"httpCode","kind":"externalParam"},{"kind":"text","text":": "},{"preciseIdentifier":"s:Si","kind":"typeIdentifier","text":"Int"},{"kind":"text","text":", "},{"kind":"externalParam","text":"cause"},{"text":": ","kind":"text"},{"kind":"typeIdentifier","preciseIdentifier":"s:s5ErrorP","text":"Error"},{"text":", ","kind":"text"},{"text":"headers","kind":"externalParam"},{"text":": [","kind":"text"},{"text":"AnyHashable","kind":"typeIdentifier","preciseIdentifier":"s:s11AnyHashableV"},{"text":" : ","kind":"text"},{"text":"Any","kind":"keyword"},{"text":"])","kind":"text"}],"type":"topic"},"doc://com.aheaditec.widgetizing.PasswordAndNfinityFramework/documentation/PasswordAndNfinityFramework/NfinityCloudError/cloudError(errorCode:errorMessage:)":{"url":"\/documentation\/passwordandnfinityframework\/nfinityclouderror\/clouderror(errorcode:errormessage:)","kind":"symbol","type":"topic","title":"NfinityCloudError.cloudError(errorCode:errorMessage:)","identifier":"doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/documentation\/PasswordAndNfinityFramework\/NfinityCloudError\/cloudError(errorCode:errorMessage:)","fragments":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"cloudError"},{"text":"(","kind":"text"},{"kind":"externalParam","text":"errorCode"},{"text":": ","kind":"text"},{"kind":"typeIdentifier","preciseIdentifier":"s:SS","text":"String"},{"kind":"text","text":"?, "},{"text":"errorMessage","kind":"externalParam"},{"text":": ","kind":"text"},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"text":"?)","kind":"text"}],"role":"symbol","abstract":[{"type":"text","text":"Represents an error with a specific error code and message that the Nfinity Cloud might return, but not from network call."}]},"doc://com.aheaditec.widgetizing.PasswordAndNfinityFramework/documentation/PasswordAndNfinityFramework/NfinityCloudError/cloudHTTPError(httpCode:response:)":{"role":"symbol","url":"\/documentation\/passwordandnfinityframework\/nfinityclouderror\/cloudhttperror(httpcode:response:)","type":"topic","title":"NfinityCloudError.cloudHTTPError(httpCode:response:)","abstract":[{"type":"text","text":"Represents an HTTP error from the Nfinity Cloud with the associated HTTP status code and a custom response object."}],"kind":"symbol","fragments":[{"text":"case","kind":"keyword"},{"kind":"text","text":" "},{"text":"cloudHTTPError","kind":"identifier"},{"text":"(","kind":"text"},{"text":"httpCode","kind":"externalParam"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"Int","preciseIdentifier":"s:Si"},{"text":", ","kind":"text"},{"kind":"externalParam","text":"response"},{"text":": ","kind":"text"},{"text":"NfinityCloudErrorResponse","preciseIdentifier":"s:27PasswordAndNfinityFramework0C18CloudErrorResponseV","kind":"typeIdentifier"},{"kind":"text","text":")"}],"identifier":"doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/documentation\/PasswordAndNfinityFramework\/NfinityCloudError\/cloudHTTPError(httpCode:response:)"},"doc://com.aheaditec.widgetizing.PasswordAndNfinityFramework/documentation/PasswordAndNfinityFramework/NfinityCloudError":{"url":"\/documentation\/passwordandnfinityframework\/nfinityclouderror","navigatorTitle":[{"kind":"identifier","text":"NfinityCloudError"}],"title":"NfinityCloudError","kind":"symbol","identifier":"doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/documentation\/PasswordAndNfinityFramework\/NfinityCloudError","fragments":[{"text":"enum","kind":"keyword"},{"kind":"text","text":" "},{"kind":"identifier","text":"NfinityCloudError"}],"abstract":[{"type":"text","text":"Represents an enum hierarchy for errors related to the NfinityÂ Cloud."}],"role":"symbol","type":"topic"},"doc://com.aheaditec.widgetizing.PasswordAndNfinityFramework/s8SendableP":{"type":"unresolvable","identifier":"doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/s8SendableP","title":"Swift.Sendable"},"doc://com.aheaditec.widgetizing.PasswordAndNfinityFramework/documentation/PasswordAndNfinityFramework/NfinityCloudError/attemptExceeded":{"title":"NfinityCloudError.attemptExceeded","type":"topic","role":"symbol","abstract":[{"text":"Case represents too many tries on ","type":"text"},{"isActive":true,"type":"reference","identifier":"doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/documentation\/PasswordAndNfinityFramework\/FaceDetectionSender\/sendFaceDetection(imageData:)"},{"text":".","type":"text"}],"kind":"symbol","identifier":"doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/documentation\/PasswordAndNfinityFramework\/NfinityCloudError\/attemptExceeded","url":"\/documentation\/passwordandnfinityframework\/nfinityclouderror\/attemptexceeded","fragments":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"attemptExceeded"}]},"doc://com.aheaditec.widgetizing.PasswordAndNfinityFramework/s5ErrorP":{"type":"unresolvable","identifier":"doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/s5ErrorP","title":"Swift.Error"}}}